lib/cisco_node_utils/client/nxapi/client.rb

```diff
@@ -186,6 +186,7 @@ def req(type, command_or_list)
     debug("Sending HTTP request to NX-API at #{@http.address}:\n" \
           "#{request.to_hash}\n#{request.body}")
     begin
+      @http.use_ssl = false
```

Comment:
Please add a comment to this to indicate why it's here, ie. "avoids EOFError from net/protocol on some platforms"